### PR TITLE
[Refactor][doc] add data-table doc

### PR DESCRIPTION
**Pull request by smallhibiscus**
# Proposed changes

Issue Number: close #xxx

## Problem Summary:

add  data-table/advance-usage\basic-usage\hit-the-rollup\data-partition  en and zh-CN docs.

## Checklist(Required)

1. Does it affect the original behavior: (Yes/No/I Don't know)
2. Has unit tests been added: (Yes/No/No Need)
3. Has document been added or modified: (Yes/No/No Need)
4. Does it need to update dependencies: (Yes/No)
5. Are there any changes that cannot be rolled back: (Yes/No)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: new-docs/en/data-table/advance-usage.md [hunk 1]
```diff
@@ -28,9 +28,9 @@ under the License.
 
 Here we introduce some of Doris's advanced features.
 
-## Table 1 Structural Change
+## Table  Structural Change
```
Comment (EmmyMiao87) [inline new-docs/en/data-table/advance-usage.md:31]:
```suggestion
## Table Structural Change
```

File: new-docs/en/data-table/data-partition.md [hunk 2]
```diff
@@ -209,9 +212,46 @@ It is also possible to use only one layer of partitioning. When using a layer pa
             
             > A new void appeared: [2017-01-01, 2017-02-01)
         
+
     In summary, the deletion of a partition does not change the scope of an existing partition. There may be holes in deleting partitions. When a partition is added by the `VALUES LESS THAN` statement, the lower bound of the partition immediately follows the upper bound of the previous partition.
+
+    In addition to the single-column partitioning we have seen above, Range partition also supports **multi-column partitioning**, examples are as follows:
+
+    ~~~text
+    ```
```
Comment (EmmyMiao87) [inline new-docs/en/data-table/data-partition.md:221]:
```suggestion
    ```text
```

File: new-docs/en/data-table/data-partition.md [hunk 3]
```diff
@@ -209,9 +212,46 @@ It is also possible to use only one layer of partitioning. When using a layer pa
             
             > A new void appeared: [2017-01-01, 2017-02-01)
         
+
     In summary, the deletion of a partition does not change the scope of an existing partition. There may be holes in deleting partitions. When a partition is added by the `VALUES LESS THAN` statement, the lower bound of the partition immediately follows the upper bound of the previous partition.
+
+    In addition to the single-column partitioning we have seen above, Range partition also supports **multi-column partitioning**, examples are as follows:
+
+    ~~~text
```
Comment (EmmyMiao87) [inline new-docs/en/data-table/data-partition.md:220]: Remove it

File: new-docs/en/data-table/data-partition.md [hunk 4]
```diff
@@ -244,7 +284,38 @@ It is also possible to use only one layer of partitioning. When using a layer pa
             p_uk: ("London")
             ```
 
-    You cannot add partitions with overlapping ranges.
+    List partition also supports **multi-column partition**, examples are as follows:
+
+    ~~~text
```
Comment (EmmyMiao87) [inline new-docs/en/data-table/data-partition.md:289]: same as above

File: new-docs/en/data-table/hit-the-rollup.md [hunk 5]
```diff
@@ -26,20 +26,164 @@ under the License.
...
+| 10002   | 2017-10-02 | 2017-10-02 13:10:00 | Shanghai  | 20   | 1    | 2017-10-02 12:59:12 | 200  | 5                | 5                |
+| 10003   | 2017-10-02 | 2017-10-02 13:15:00 | Guangzhou | 32   | 0    | 2017-10-02 11:20:00 | 30   | 11               | 11               |
+| 10004   | 2017-10-01 | 2017-10-01 12:12:48 | Shenzhen  | 35   | 0    | 2017-10-01 10:00:15 | 100  | 3                | 3                |
+| 10004   | 2017-10-03 | 2017-10-03 12:38:20 | Shenzhen  | 35   | 0    | 2017-10-03 10:20:22 | 11   | 6                | 6                |
+
+On this basis, we create a ROLLUP:
+
+| ColumnName |
+| ---------- |
+| user_id    |
```
Comment (EmmyMiao87) [inline new-docs/en/data-table/hit-the-rollup.md:78]:
```suggestion
| user\_id    |
```

File: new-docs/en/data-table/hit-the-rollup.md [hunk 6]
```diff
@@ -26,20 +26,164 @@ under the License.
...
+| age             | SMALLINT     |                 | User age                               |
+| sex             | TINYINT      |                 | User gender                            |
+| Last_visit_date | DATETIME     | REPLACE         | Last user access time                  |
+| Cost            | BIGINT       | SUM             | Total User Consumption                 |
+| max dwell time  | INT          | MAX             | Maximum user residence time            |
+| min dwell time  | INT          | MIN             | User minimum residence time            |
+
+The data stored are as follows:
+
+| user_id | date       | timestamp           | city      | age  | sex  | last\_visit\_date   | cost | max\_dwell\_time | min\_dwell\_time |
```
Comment (EmmyMiao87) [inline new-docs/en/data-table/hit-the-rollup.md:64]:
```suggestion
| user\_id | date       | timestamp           | city      | age  | sex  | last\_visit\_date   | cost | max\_dwell\_time | min\_dwell\_time |
```

File: new-docs/en/data-table/hit-the-rollup.md [hunk 7]
```diff
@@ -26,20 +26,164 @@ under the License.
...
+
+Because Uniq is only a special case of the Aggregate model, we do not distinguish it here.
+
+Example 1: Get the total consumption per user
+
+Following [Data Model Aggregate Model](data-model.html#Aggregate Model) in the **Aggregate Model** section, the Base table structure is as follows:
+
+| ColumnName      | Type         | AggregationType | Comment                                |
+| --------------- | ------------ | --------------- | -------------------------------------- |
+| user_id         | LARGEINT     |                 | user id                                |
```
Comment (EmmyMiao87) [inline new-docs/en/data-table/hit-the-rollup.md:51]:
```suggestion
| user\_id         | LARGEINT     |                 | user id                                |
```

File: new-docs/en/data-table/hit-the-rollup.md [hunk 8]
```diff
@@ -26,20 +26,164 @@ under the License.
...
+| 10004   | 2017-10-03 | 2017-10-03 12:38:20 | Shenzhen  | 35   | 0    | 2017-10-03 10:20:22 | 11   | 6                | 6                |
+
+On this basis, we create a ROLLUP:
+
+| ColumnName |
+| ---------- |
+| user_id    |
+| cost       |
+
+The ROLLUP contains only two columns: user_id and cost. After the creation, the data stored in the ROLLUP is as follows:
```
Comment (EmmyMiao87) [inline new-docs/en/data-table/hit-the-rollup.md:81]:
```suggestion
The ROLLUP contains only two columns: user\_id and cost. After the creation, the data stored in the ROLLUP is as follows:
```

File: new-docs/en/data-table/hit-the-rollup.md [hunk 9]
```diff
@@ -26,20 +26,164 @@ under the License.
...
+
+| user\_id | cost |
+| -------- | ---- |
+| 10000    | 35   |
+| 10001    | 2    |
+| 10002    | 200  |
+| 10003    | 30   |
+| 10004    | 111  |
+
+As you can see, ROLLUP retains only the results of SUM on the cost column for each user_id. So when we do the following query:
```
Comment (EmmyMiao87) [inline new-docs/en/data-table/hit-the-rollup.md:91]:
```suggestion
As you can see, ROLLUP retains only the results of SUM on the cost column for each user\_id. So when we do the following query:
```

File: new-docs/en/data-table/hit-the-rollup.md [hunk 10]
```diff
@@ -26,20 +26,164 @@ under the License.
...
+
+| ColumnName       | Type         |
+| ---------------- | ------------ |
+| age              | INT          |
+| user\_id         | BIGINT       |
+| message          | VARCHAR(100) |
+| max\_dwell\_time | DATETIME     |
+| min\_dwell\_time | DATETIME     |
+
+As you can see, the columns of ROLLUP and Base tables are exactly the same, just changing the order of user_id and age. So when we do the following query:
```
Comment (EmmyMiao87) [inline new-docs/en/data-table/hit-the-rollup.md:157]:
```suggestion
As you can see, the columns of ROLLUP and Base tables are exactly the same, just changing the order of user\_id and age. So when we do the following query:
```

**Comment by smallhibiscus** [comment]
Thank you,I have modified these.

**Review by hf200012 — APPROVED** [review]
LGTM

**Pull request merged**